nautobot_golden_config/models.py

```diff
@@ -237,8 +298,18 @@ class ComplianceRule(PrimaryModel):  # pylint: disable=too-many-ancestors
         "match_config",
         "config_type",
         "custom_compliance",
+        "config_remediation",
     ]
 
+    @property
+    def remediation_setting(self):
+        """Returns remediation settings for a particular platform."""
+        try:
+            remediation_setting = RemediationSetting.objects.get(platform=self.platform)
+        except Exception as err:  # pylint: disable=broad-except:
+            raise ValidationError(f"Platform {self.platform.slug} has no Remediation Settings defined.") from err
```

Comment:
This doesn't seem scoped enough there should be an explicit exception that we should catch.

Reply:
> This doesn't seem scoped enough there should be an explicit exception that we should catch.

DONE. Narrowed it down to model.DoesNotExist.